py3status/modules/xrandr_rotate.py

```diff
@@ -29,6 +29,13 @@
     {icon} a rotation icon, specified by `horizontal_icon` or `vertical_icon`.
 
 
+Remarks:
+    We have seen a case when rotating a screen using this module crashed i3 session.
```

Comment:
I'd change this to something like "there have been cases where this causes usability issues and make break your i3 session.  But for most users it works great", as trying to explain the issue is hard. i3 doesn't crash but becomes unusable till sending SIGCONT to the xrandr process via a vterm is just too complicated to explain easily. Or maybe add `Experimental` @ultrabug may have a view.